src/Features/Core/Portable/ImplementAbstractClass/AbstractImplementAbstractClassService.Editor.cs

```diff
@@ -153,13 +153,7 @@ private partial class Editor
                     setMethod: setMethod);
             }
 
-            private bool ShouldGenerateAccessor(IMethodSymbol method)
-            {
-                return
-                    method != null &&
-                    method.IsAccessibleWithin(_state.ClassType) &&
-                    _state.ClassType.FindImplementationForAbstractMember(method) == null;
```

Comment:
Once we've created the state object, we've already identified which members need to be implemented, see [here](https://github.com/dotnet/roslyn/blob/7755b1632509f7fc9c6f494c99711dae8d700499/src/Features/Core/Portable/ImplementAbstractClass/AbstractImplementAbstractClassService.State.cs#L55)

Unless I am missing something, there should be no case where we want to generate a property, but the properties accessor are already implemented.

What the code used to say was:

"Don't generate an accessor if it is not required by the interface/class, is inaccessible, or it already exists somehow, **But Still Generate The Propery Anyway**"

Now we say:
"Since we've already decided to generate this property by getting to this location in the code, generate an accessor unless it is not required by the interface/class."